docs/quickstart/queue-a.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: armadaproject.io/v1
+kind: Queue
+name: queue-a
+permissions:
```

Comment:
@severinson it was mine concern as well. I've had a discussion with @robertdavidsmith and @JamesMurkin and for current requirements it was concluded that this should suffice.

Reply:
All right. I'm still concerned since the file format becomes part of Armada's API. If other systems, or worse, customers (although I don't think we have customers that can create queues?) become dependent on it, changing it may be difficult. If we're merging this as a temporary measure, I suggest adding comments to the related `.proto` and `.go` files saying something like `This API is not considered stable and may be changed without warning at any time.`

Reply:
The main feature introduced here is adding ability to create any Armada resource through `armadactl create -f file_path` command. Old command for creating queues stays the same, and all existing functionalities have unchanged behaviour. It expands current owners system to be more granular when needed. Regarding stability I'm not sure how it should be treated. @robertdavidsmith @JamesMurkin

Reply:
Just saying it's a nice habit to add a comment to parts of the API that we don't consider to be stable.